src/sentry/notifications/notification_action/action_validation.py

```diff
@@ -206,31 +206,70 @@ def __init__(self, validated_data: dict[str, Any], organization: Organization) -
         self.validated_data = validated_data
         self.organization = organization
 
+    def _get_sentry_app_installation(
+        self, sentry_app_identifier: SentryAppIdentifier, target_identifier: str
+    ) -> RpcSentryAppInstallation | None:
+        """
+        Get the sentry app installation based on whether the target identifier is an installation id or sentry app id
+        We do not want to accept SentryAppIdentifier.SENTRY_APP_INSTALLATION_UUID long term, this is temporary until we migrate the data over
+        """
+        installations = None
+        installation = None
```

Comment:
Is typing gonna complain this needs to be `RpcSentryAppInstallation | None = None`?